### PR TITLE
Extend Live Examples by complementary palette

### PR DESCRIPTION
**Pull request by scavnickyj**
- add complementary palette to customTheme used by themed examples
- overwrite several css variables inside ExampleWithSource in order to avoid theming in source part
- remove themed RankingFilter example as it is not fully supported by complementary palette yet
- add themed BarChart example

Screenshot 2021-04-01 at 5 15 54 PM

JIRA: BB-2888

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3082/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4732/ : SUCCESS in 12m 23s

File: examples/sdk-examples/src/components/SourceContainer.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export const SourceContainer: React.FC<ISourceContainerProps> = ({ isJS, toggleI
 
                     .gd-tab.is-active {
                         color: #ffffff;
+                        border-color: #14b2e2;
```
Comment (scavnickyj) [inline examples/sdk-examples/src/components/SourceContainer.tsx:46]: Fixing initial color to avoid using css variables.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3083/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4733/ : SUCCESS in 13m 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3084/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4734/ : SUCCESS in 12m 29s

File: examples/sdk-examples/src/components/SourceContainer.tsx (hunk 1 above)
Comment (scavnickyj) [reply examples/sdk-examples/src/components/SourceContainer.tsx:47]: Fixing font family

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3092/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4741/ : SUCCESS in 12m 30s

**Comment by strouhaljan** [comment]
LGTM

**Review by strouhaljan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1211

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/922/ : SUCCESS in 53s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1190/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4742/ : SUCCESS in 12m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/304/ : SUCCESS in 8m 15s (non-voting)

**Pull request merged**